### PR TITLE
[sv-SE] Updated strings to match the format of 4.3

### PR DESCRIPTION
**Pull request by tangix**
Updated the strings to match quote-style introduced in 4.3 by https://github.com/codeigniter4/CodeIgniter4/pull/6474

### CODE REVIEW

**Comment by tangix** [comment]
Hmmm.... since this PR is pretty old I wonder if I should rebase this on a more recent version and open a new PR.

File: Language/sv-SE/Session.php [hunk 1]
```diff
@@ -14,6 +14,6 @@
     'invalidSavePath'        => 'Session: Angivna sökvägen "{0}" är inte en katalog, existerar inte eller kan inte skapas.',
     'writeProtectedSavePath' => 'Session: Angivna sökvägen "{0}" är inte skrivbar för PHP processen.',
     'emptySavePath'          => 'Session: Ingen sökväg har angetts.',
-    'invalidSavePathFormat'  => 'Session: Ogiltigt format för Redis: {0}',
-    'invalidSameSiteSetting' => 'Session: SameSite måste vara None, Lax, Strict, eller en blank sträng. Givet: {0}',
+    'invalidSavePathFormat'  => 'Session: Ogiltig sökväg för Redis: "{0}"',
```
Comment (MGatner) [inline Language/sv-SE/Session.php:17]: Was this a mistake before?

**Review by MGatner — APPROVED** [review]
Looks good to me, without knowing the language.

File: Language/sv-SE/Session.php (hunk 1 above)
Comment (paulbalandan) [reply Language/sv-SE/Session.php:17]: @tangix you can open a new PR if that would be faster for you.

**Comment by tangix** [comment]
Scratch this PR. I'm trying to create a new one.

**Pull request closed without merging**